datasets.py

```diff
@@ -117,6 +122,89 @@ def read_xml_content(xml_file):
     11: 'indoor'
 }
 
+def group_mapping_creator(labels_to_names, supercategories_to_names, 
```

Comment:
make the default for supercategories_to_names if none is provided, GROUPINGS_TO_NAMES

Reply:
done, thanks